routers/repo/pull.go

```diff
@@ -454,6 +454,18 @@ func MergePullRequest(ctx *context.Context) {
 
 	pr.Issue = issue
 	pr.Issue.Repo = ctx.Repo.Repository
+
+	noDeps, err := models.IssueNoDependenciesLeft(issue)
+	if err != nil {
```

Comment:
Please minimize that as shown above

Reply:
Don't think it is worth it, I would need to declare `noDeps` before calling the function which wouldn't result in better readability.